### PR TITLE
AWS Batch Bug Fixes

### PR DESCRIPTION
**Pull request by dxenes1**
Fixes bugs with pool and s3wrap. Adds an example cell detect workflow using batch.

### CODE REVIEW

File: conduit/config/pool_config.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "Local": {
+        "description": "Local Execution Pool",
+        "slots": 100
```
Comment (erikjohnson24) [inline conduit/config/pool_config.json]: I would make the pool size much smaller here by default- maybe 4?

File: conduit/utils/cwlparser.py [hunk 2]
```diff
@@ -232,7 +232,8 @@ def create_subdag(self, iteration, i, param_db_update_dict, job_params, job, wf_
                     dag=subdag,
                     queue=step_job_queue,
                     workflow_id=parent_dag_id,
-                    score_format=score_format
+                    score_format=score_format,
+                    pool='Batch'
```
Comment (erikjohnson24) [inline conduit/utils/cwlparser.py]: if cwl doc is local, why is the pool batch?

File: saber/xbrain/jobs/cell_detect/batch.cwl [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    use_adam: boolean
+    learning_rate: float
+    decay: float
+    momentum: float
+    beta1: float
+    beta2: float
+    save_freq: int
+    do_warp: boolean
+    tile_size: int
+#    weights_file: File?
```
Comment (erikjohnson24) [inline saber/xbrain/jobs/cell_detect/batch.cwl]: clean up and remove commented params

File: conduit/utils/cwlparser.py (hunk 2 above)
Comment (dxenes1) [reply conduit/utils/cwlparser.py]: This might be github showing the changes weird. The `pool=Batch` line is in the AWSBatchOperator section. (Line 236)

**Pull request merged**